<?xml version="1.0"?>

<launch>

  <node pkg="husky_highlevel_controller" type="husky_highlevel_controller" name="husky_highlevel_controller_node" output="screen">
    <rosparam command="load" file="$(find husky_highlevel_controller)/config/Config.yaml"/>
  </node>
  <!--node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="teleop_twist_keyboard" output="screen"/-->

  <arg name="world_name" default="$(find husky_highlevel_controller)/worlds/singlePillar.world"/>
  <!-- arg name="world_name" default="worlds/robocup14_spl_field.world"/-->
  <arg name="laser_enabled" default="true"/>
  <include file="$(find husky_gazebo)/launch/husky_empty_world.launch">
    <arg name="world_name" value="$(arg world_name)"/>
    <!--arg name="world_name" value="$(find husky_highlevel_controller)/worlds/singlePillar.world"/-->
    <arg name="laser_enabled" value="$(arg laser_enabled)"/>
  </include>
<!-- 在rviz中显示-->
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find husky_highlevel_controller)/config/laserConfig.rviz" required="true" />
	


</launch>
